crates/bip39/src/mnemonic.rs

```diff
@@ -132,11 +132,11 @@ impl Entropy {
     /// not a valid entropy length
     pub fn from_rng<R: Rng>(bytes: usize, rng: &mut R) -> Result<Self, MnemonicError> {
```

Comment:
because `R: Rng` is part of the public API here, this is technically semver major?